cmd/cli/devstack/devstack.go

```diff
@@ -166,7 +167,7 @@ func runDevstack(cmd *cobra.Command, ODs *devstack.DevStackOptions, IsNoop bool)
 	}
 
 	// make sure we don't run devstack with a custom IPFS path - that must be used only with serve
-	if config.GetServeIPFSPath() != "" {
+	if path, err := config.Get[string](types.NodeIPFSServePath); err == nil && path != "" {
```

Comment:
I think we should return the error if it's not nil as it could indicate some malformed config. Otherwise looks great.